src/azure-cli/azure/cli/command_modules/appservice/custom.py

```diff
@@ -2354,39 +2355,61 @@ def delete_ssl_cert(cmd, resource_group_name, certificate_thumbprint):
     raise CLIError("Certificate for thumbprint '{}' not found".format(certificate_thumbprint))
 
 
-def import_ssl_cert(cmd, resource_group_name, name, key_vault, key_vault_certificate_name):
+def import_ssl_cert(cmd, resource_group_name, name, certificate_name=None, key_vault_certificate_name=None,
+                    key_vault=None):
     Certificate = cmd.get_models('Certificate')
     client = web_client_factory(cmd.cli_ctx)
     webapp = client.web_apps.get(resource_group_name, name)
     if not webapp:
         raise CLIError("'{}' app doesn't exist in resource group {}".format(name, resource_group_name))
     server_farm_id = webapp.server_farm_id
     location = webapp.location
-    kv_id = None
-    if not is_valid_resource_id(key_vault):
```

Comment:
Why are we deleting this?

Reply:
moved to helper function get_import_kv_details